src/Tizen.NUI/src/internal/Application/Application.cs

```diff
@@ -1030,6 +1084,47 @@ private void OnNUIApplicationMemoryLow(MemoryStatus status)
             applicationMemoryLowEventHandler?.Invoke(this, e);
         }
 
+        /**
+          * @brief Event for changing Device orientation signal which can be used to subscribe/unsubscribe the event handler
```

Comment:
please remove @brief and please use // for comments

Reply:
I see, I will update all of cases in this file.